testutil/testclient/testqueryclients/appquerier.go

```diff
@@ -0,0 +1,86 @@
+package testqueryclients
+
+import (
+	"context"
+	"testing"
+
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/golang/mock/gomock"
+
+	"github.com/pokt-network/poktroll/testutil/mockclient"
+	"github.com/pokt-network/poktroll/x/application/types"
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+)
+
+// addressApplicationMap is a map of application addresses that are deemed to
+// exist on chain, if an address is not in this map an error will be returned
+// from the mock ApplicationQueryClient's GetApplication method.
+// The array of strings is the addresses of the delegated gateways that the
+// application is supposed to have.
```

Comment:
```suggestion
// addressApplicationMap is a map of:
// 	applicationAddresses -> list of delegated gateways by app.
// All addresses are assumed to exist on chain.
```

Reply:
1. I think the error piece is good but doesn't relate to the map
2. Sometimes, less words and more pseudo code in the comments is easier for future developers to read.